### PR TITLE
**Fix:** Small fixes for accordion and tabs

### PR DESCRIPTION
**Pull request by stereobooster**
# Summary

- Add hover for plus
- Made icons in accordion unfocusable
- Fix warning in the console

# Related issue

# To be tested

Me
- [ ] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "scripts": {
     "start": "ts-node -P tsconfig.node.json ./scripts/start.ts",
     "build": "run-s build:*",
+    "clean:icons": "ts-node -P tsconfig.node.json ./scripts/build-icons.ts -c ./src/Icon",
```
Comment (fabien0102) [inline package.json]: I don't see how this can works, you missed the `-o` option, personnally I just do `yarn build:icons -c` when I need a clean build 😉

**Comment by fabien0102** [comment]
Can you also add some other littles fixes to the tabs here?

The specs for this:
> Let’s improve the hover style of the “X” for closing a tab. It should always be the same color (whether tab is active or inactive): #BFCBD2. The circle around the X should be 24px not 30px (it’s too big right now).

File: package.json (hunk 1 above)
Comment (stereobooster) [reply package.json]: I had no idea

**Comment by fabien0102** [comment]
https://app.zeplin.io/project/5cc31fad6f17562d6dac9a53/screen/5d42f85de37a238930a05444

**Pull request merged**